spec/services/browser_support_spec.rb

```diff
@@ -146,6 +146,46 @@
           it { expect(supported).to eq(true) }
         end
       end
+
+      context 'with opera desktop user agent' do
+        let(:user_agent) do
+          # Opera 83
+          'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) ' \
+            'Chrome/103.0.5046.0 Safari/537.36 OPR/83.0.4254.70'
+        end
+
+        context 'with version within opera mobile supported range, but no support for desktop' do
+          before do
+            allow(BrowserSupport).to receive(:browser_support_config).and_return(['op_mob 73'])
+          end
+
+          it { expect(supported).to eq(false) }
+        end
+      end
+
+      context 'with opera mobile user agent' do
+        let(:user_agent) do
+          # Opera Mobile 72
+          'Mozilla/5.0 (Linux; Android 6.0.1; TX2) AppleWebKit/537.36 (KHTML, like Gecko) ' \
+            'Chrome/106.0.5249.126 Safari/537.36 OPR/72.9.3767.72992'
+        end
+
+        context 'with version within supported range' do
+          before do
+            allow(BrowserSupport).to receive(:browser_support_config).and_return(['op_mob 72'])
+          end
+
+          it { expect(supported).to eq(true) }
+        end
+
+        context 'with version below supported range' do
+          before do
+            allow(BrowserSupport).to receive(:browser_support_config).and_return(['op_mob 73'])
```

Comment:
73 is higher than 72? is should it be "higher than supported range" or should the number be lower?

Reply:
I'll adjust this, since I took a slightly inconsistent approach with how I structured these `context`s vs. elsewhere in the file. In this case it was meant to say that the supported version is 73, and the version (which is derived from the `user_agent` above as `72`) is "below" that supported range.

Elsewhere we use a consistent `browser_support_config` and these individual `context` will change the user agent, so I'll adjust to that and hopefully that'll make it a little clearer.

Reply:
Updated in e2cbd9a93.